### PR TITLE
feat: i18n module

### PR DESCRIPTION
**Pull request by mortennordseth**
This PR adds `next-intl` for language support. The only language supported for now is `no`. Since this is the only supported language I didn't spend time creating a smart `getUserLocale` function. Later, if we add more languages, this should support browser locale. 

Closes https://github.com/mrfylke/kontraktsportal-org/issues/24

### CODE REVIEW

**Review by adriansberg — APPROVED** [review]
This looks good! `next-intl` looks simple to use 👏

File: src/modules/i18n/request.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { getRequestConfig } from 'next-intl/server';
+import { getUserLocale } from './locale';
+
+export default getRequestConfig(async () => {
+  const locale = await getUserLocale();
+
+  return {
+    locale,
+    messages: (await import(`./translations/${locale}.json`)).default,
```
Comment (mikaelbr) [inline src/modules/i18n/request.ts:9]: Is this recommended way to do it by the module?

Comment (mortennordseth) [reply src/modules/i18n/request.ts:9]: Yes: https://next-intl-docs.vercel.app/docs/getting-started/app-router/without-i18n-routing#i18n-request

Comment (mikaelbr) [reply src/modules/i18n/request.ts:9]: Makes sense now that I see it is a json file

**Review by mikaelbr — APPROVED** [review]
Works like cheese on a Friday evening

**Pull request merged**